packages/transformers/posthtml/src/PostHTMLTransformer.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+
+import posthtml from 'posthtml';
+import parse from 'posthtml-parser';
+import render from 'posthtml-render';
+import nullthrows from 'nullthrows';
+import semver from 'semver';
+
+const getPostHTMLConfig = async asset => {
+  let config = await asset.getConfig(
+    ['.posthtmlrc', '.posthtmlrc.js', 'posthtml.config.js'],
+    {
+      packageKey: 'posthtml'
+    }
+  );
+  if (!config && !asset.options.minify) {
+    return;
+  }
+
+  config = config || {};
+  const plugins = config.plugins;
+  if (typeof plugins === 'object') {
+    // This is deprecated in favor of result messages but kept for compatibility
+    // See https://github.com/posthtml/posthtml-include/blob/e4f2a57c2e52ff721eed747b65eddf7d7a1451e3/index.js#L18-L26
+    const depConfig = {
+      addDependencyTo: {
+        addDependency: name =>
+          asset.addDependency(name, {includedInParent: true})
+      }
+    };
+    Object.keys(plugins).forEach(p => Object.assign(plugins[p], depConfig));
+  }
+  // TODO: find a way to load plugins
+  // config.plugins = await loadPlugins(plugins, asset.name);
```

Comment:
hmm not sure if we should try to make https://github.com/parcel-bundler/parcel/blob/425885992156b6b3e3ca02b3ffd1693fff6cbba5/packages/transformers/postcss/src/loadPlugins.js#L6 more generic or not

Reply:
They were shared before but I was hesitate and didn't see postcss was converted yet, so I decided to hold off on it. However I also realized this is probably why the tests are failing. 😅